--- /home/ivan/root/data/software/src/freebsd/src/sys/kern/kern_sig.c	2015-08-12 17:22:05.000000000 +0300
+++ kern_sig.c	2016-10-02 01:22:21.321618140 +0300
@@ -37,6 +37,7 @@
 #include <sys/cdefs.h>
 __FBSDID("$FreeBSD: releng/10.2/sys/kern/kern_sig.c 284343 2015-06-13 16:15:43Z jhb $");
 
+#include "opt_sched.h"
 #include "opt_compat.h"
 #include "opt_kdtrace.h"
 #include "opt_ktrace.h"
@@ -1713,6 +1714,13 @@
 	struct proc *p;
 	int error;
 
+#ifdef SCHED_U
+	if( (u_int)uap->signum >= SIG_SCHEDMIN 
+	&&	(u_int)uap->signum <= SIG_SCHEDMAX ){
+		sched_control(uap->signum, td);
+		return 0;
+	}
+#endif
 	/*
 	 * A process in capability mode can send signals only to himself.
 	 * The main rationale behind this is that abort(3) is implemented as
